### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by walstork**
Updated unit test files

### CODE REVIEW

File: integrations/gitlab/gitlab_test.go [hunk 1]
```diff
@@ -234,14 +227,7 @@ func TestOnboard(t *testing.T) {
...
 
-	errors := make(chan error, 1)
-
-	p := &progress{
-		integration: i,
-		wg:          &ewg,
-		errors:      errors,
-		logger:      logger,
-	}
+	//errors := make(chan error, 1)
```
Comment (josecordaz) [inline integrations/gitlab/gitlab_test.go]: please, delete this comment, if you don't need it.

Comment (walstork) [reply integrations/gitlab/gitlab_test.go]: Done

**Comment by xzandrew** [comment]
I've just run this, there are still multiple test failures. Maybe a good idea to review which tests are worth fixing and if some should be just removed.  Would probably have to do the same for europa branch.

**Comment by jhaynie** [comment]
i'm thinking it would be more beneficial to just focus on the europa branch and make sure 100% of the unit tests pass (even if you need a setup, which needs to be documented) and enable the CI build on CircleCI. We're going to be merging europa to master soon and i don't want to have to keep having backport merge conflicts each time we merge master into europa.

**Pull request closed without merging**